bridge/client/app/_components/ktb-task-item/ktb-task-item.component.html

```diff
@@ -41,6 +41,7 @@ <h4 class="m-0 mt-1 mb-1" [class.error]="task.isFaulty()" [textContent]="task.ge
             [overlayDisabled]="true"
           ></ktb-evaluation-info>
         </div>
+        <div *ngIf="task.traces.length === 0" class="mt-2">No events are available for this sequence.</div>
```

Comment:
![image](https://user-images.githubusercontent.com/6098219/151125925-e4e65f40-ba68-4a29-9344-976ac10ed901.png)

We are showing now this message for sequences and for tasks. Can we differentiate here?

Reply:
Added a isSubtask flag, if set then the message is not displayed